modules/apps/portal-remote/portal-remote-cors-impl/src/main/java/com/liferay/portal/remote/cors/internal/BaseTrieURLToCORSSupportMapper.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.remote.cors.internal;
+
+import com.liferay.portal.kernel.util.Validator;
+
+/**
+ * @author Carlos Sierra Andrés
+ * @author Arthur Chan
+ */
+public abstract class BaseTrieURLToCORSSupportMapper
+	extends BaseURLToCORSSupportMapper {
+
+	@Override
+	public CORSSupport get(String urlPath) {
+		CORSSupport corsSupport = getWildcardCORSSupport(urlPath);
+
+		if (corsSupport != null) {
+			return corsSupport;
+		}
+
+		return getExtensionCORSSupport(urlPath);
+	}
```

Comment:
you have lost the try/catch block added here? https://github.com/liferay-appsec/liferay-portal/pull/78/commits/70963af07eeaf3d1c1f79e45bcb22bd5f1477acf#diff-1382af0175893d40352138b6ea878977R59

Reply:
Sorry, will do.